cli/docker/Dockerfile

```diff
@@ -15,6 +15,18 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
+# The source code for the binaries installed to the Docker image as part of this
+# Dockerfile can be downloaded from the following locations.
+#
+# - For the base image: https://github.com/docker-library/openjdk/blob/master/11/jre/slim/Dockerfile
```

Comment:
Not a fan of starting each sentence with 'for' think we can do better. @adambhere  What do you think?

Reply:
Consider this version instead (slightly simpler and less repetitious):

# The source code for the binaries installed to the Docker image as part of this
# Dockerfile can be downloaded from the locations listed below.
#
# - Base image: https://github.com/docker-library/openjdk/blob/master/11/jre/slim/Dockerfile
# - Debian Sid packages: https://sources.debian.org/
# - git-repo tool: https://gerrit-review.googlesource.com/admin/repos/git-repo
# - NPM packages: https://www.npmjs.com/
# - Python packages: https://pypi.org/
# - Go language's dep tool: https://github.com/golang/dep
# - Haskell language's stack tool: https://github.com/commercialhaskell/stack
# - ScanCode tool: https://github.com/nexB/scancode-toolkit

Reply:
Consider this:

```
# The source code for the binaries installed to the Docker image as part of this
# Dockerfile can be downloaded from the locations listed below.
#
# - Base image: https://github.com/docker-library/openjdk/blob/master/11/jre/slim/Dockerfile
# - Debian Sid packages: https://sources.debian.org/
# - git-repo tool: https://gerrit-review.googlesource.com/admin/repos/git-repo
# - NPM packages: https://www.npmjs.com/
# - Python packages: https://pypi.org/
# - Go language's dep tool: https://github.com/golang/dep
# - Haskell language's stack tool: https://github.com/commercialhaskell/stack
# - ScanCode tool: https://github.com/nexB/scancode-toolkit
```